### DISCUSSION TITLE
Can I use NEB images and/or geo opt results to train DeePMD parameters?

### DISCUSSION THREAD

**Discussion by turbosonics** [deepmodeling/deepmd-kit#2689] [Q&A]
Hello.

I'm learning this tool by following tutorials. But just in case I'm asking, can I include NEB images to the training set of DeePMD?

For example, let's assume I have some AIMD results of crystal structure of interest from VASP or QE. And then I operated five NEB searches for specific cation-vacancy long range diffusion, generated 20 images for five diffusion pathways. That is total 100 images for 5 pathways, and each image have xyz coordinates of atoms, energies, and forces.

I hope my NNFF to describe this cation kinetics and diffusion really nicely, so I hope to combine AIMD results and NEB results in the same training set for the DeePMD parameters of this crystal. Is this possible for DeePMD? 

If not, what would be DFT suggestion to get the diffusion pathway correctly for DeePMD training?

Similarly, if I conduct some bond distance scan or bond angle scan (multiple constrained geometry optimizations) from DFT, can I include them in the training set with AIMD results for DeePMD training? I don't know if DeePMD could read results from molecular orbital DFT tools like Gaussian. But I think I can try some "scanning" geo opts to get bond dissociation energy curve using VASP or QE.

Thank you for your answer in advance.

Best,

**Comment by njzjz** [comment]
The sampling method does not matter. The only requirement is that the DFT level should be consistent.

DeePMD-kit is not a tool to read results from different software. We have [dpdata](https://github.com/deepmodeling/dpdata) to do that, but if not supported, you are always free to prepare data in your script.
